<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>msatcommander</class>
 <widget class="QDialog" name="msatcommander">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>546</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>msatcommander-1.0.0</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>501</width>
     <height>491</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="usesScrollButtons">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>General</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>221</width>
       <height>241</height>
      </rect>
     </property>
     <property name="title">
      <string>Search Parameters</string>
     </property>
     <widget class="QCheckBox" name="dinucCheckBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>46</y>
        <width>115</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Dinucleotide</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="pentanucCheckBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>121</y>
        <width>120</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Pentanucleotide</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="trinucCheckBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>71</y>
        <width>120</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Trinucleotide</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="hexanucCheckBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>146</y>
        <width>120</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Hexanucleotide</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="tetranucCheckBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>96</y>
        <width>120</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Tetranucleotide</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="combineLociCheckBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Combine Loci</string>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>201</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QCheckBox" name="perfectRepeatsCheckBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Perfect Repeats</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="mononucCheckBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>21</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Mononucleotide</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="mononucSpinBox">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>25</y>
        <width>51</width>
        <height>25</height>
       </rect>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>10</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="dinucSpinBox">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>50</y>
        <width>51</width>
        <height>25</height>
       </rect>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>8</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="trinucSpinBox">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>75</y>
        <width>51</width>
        <height>25</height>
       </rect>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>8</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="tetranucSpinBox">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>100</y>
        <width>51</width>
        <height>25</height>
       </rect>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>6</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="pentanucSpinBox">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>125</y>
        <width>51</width>
        <height>25</height>
       </rect>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>6</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="hexanucSpinBox">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>150</y>
        <width>51</width>
        <height>25</height>
       </rect>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>6</number>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>221</width>
       <height>201</height>
      </rect>
     </property>
     <property name="title">
      <string>Primer Design</string>
     </property>
     <widget class="QCheckBox" name="designPrimersCheckBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Design Primers</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="tagPrimersCheckBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Tag Primers</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="pigtailPrimersCheckBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Pigtail Primers</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cagTagCheckBox">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>CAG Tag</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="m13rCheckBox">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>M13R Tag</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="customTagCheckBox">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>110</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Custom Tag</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="customTagLineEdit">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>130</y>
        <width>160</width>
        <height>22</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhNone</set>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="pigtailPrimersTagLineEdit">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>170</y>
        <width>160</width>
        <height>22</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhNone</set>
      </property>
      <property name="text">
       <string>GTTT</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>265</x>
       <y>0</y>
       <width>221</width>
       <height>71</height>
      </rect>
     </property>
     <property name="title">
      <string>Multiprocessing</string>
     </property>
     <widget class="QCheckBox" name="useMultiprocessingCheckBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Use Multiprocessing</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>265</x>
       <y>90</y>
       <width>221</width>
       <height>191</height>
      </rect>
     </property>
     <property name="title">
      <string>Output</string>
     </property>
     <widget class="QCheckBox" name="repeatsCheckBox">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>70</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Repeats</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="combinedRepeatsCheckBox">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>90</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Combined Repeats</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="primersCheckBox">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>110</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Primers</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="taggedPrimersCheckBox">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>130</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Tagged Primers</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="commaSeparatedRadioButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Comma-Separated</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="tabDelimitedRadioButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Tab-Delimited</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="keepDatabaseCheckBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Keep Database</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Advanced</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>191</width>
       <height>103</height>
      </rect>
     </property>
     <property name="title">
      <string>Primer Sizes</string>
     </property>
     <widget class="QSpinBox" name="spinBox_4">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>22</y>
        <width>58</width>
        <height>25</height>
       </rect>
      </property>
      <property name="minimum">
       <number>15</number>
      </property>
      <property name="maximum">
       <number>30</number>
      </property>
      <property name="value">
       <number>18</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>24</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Primer Min Size</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Primer Opt Size</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_6">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>48</y>
        <width>58</width>
        <height>25</height>
       </rect>
      </property>
      <property name="minimum">
       <number>15</number>
      </property>
      <property name="maximum">
       <number>30</number>
      </property>
      <property name="value">
       <number>18</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>76</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Primer Max Size</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_7">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>74</y>
        <width>58</width>
        <height>25</height>
       </rect>
      </property>
      <property name="minimum">
       <number>15</number>
      </property>
      <property name="maximum">
       <number>30</number>
      </property>
      <property name="value">
       <number>18</number>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>481</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title">
      <string>Product Size</string>
     </property>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>38</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Product Size</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>40</y>
        <width>371</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>191</width>
       <height>103</height>
      </rect>
     </property>
     <property name="title">
      <string>Primer Tms</string>
     </property>
     <widget class="QSpinBox" name="spinBox_5">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>22</y>
        <width>58</width>
        <height>25</height>
       </rect>
      </property>
      <property name="minimum">
       <number>15</number>
      </property>
      <property name="maximum">
       <number>30</number>
      </property>
      <property name="value">
       <number>18</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>24</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Primer Min Tm</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Primer Opt Tm</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_8">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>48</y>
        <width>58</width>
        <height>25</height>
       </rect>
      </property>
      <property name="minimum">
       <number>15</number>
      </property>
      <property name="maximum">
       <number>30</number>
      </property>
      <property name="value">
       <number>18</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>76</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Primer Max Tm</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_9">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>74</y>
        <width>58</width>
        <height>25</height>
       </rect>
      </property>
      <property name="minimum">
       <number>15</number>
      </property>
      <property name="maximum">
       <number>30</number>
      </property>
      <property name="value">
       <number>18</number>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="runButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>510</y>
     <width>115</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>510</y>
     <width>115</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>mononucCheckBox</tabstop>
  <tabstop>mononucSpinBox</tabstop>
  <tabstop>dinucCheckBox</tabstop>
  <tabstop>dinucSpinBox</tabstop>
  <tabstop>trinucCheckBox</tabstop>
  <tabstop>trinucSpinBox</tabstop>
  <tabstop>tetranucCheckBox</tabstop>
  <tabstop>tetranucSpinBox</tabstop>
  <tabstop>pentanucCheckBox</tabstop>
  <tabstop>pentanucSpinBox</tabstop>
  <tabstop>hexanucCheckBox</tabstop>
  <tabstop>combineLociCheckBox</tabstop>
  <tabstop>tagPrimersCheckBox</tabstop>
  <tabstop>designPrimersCheckBox</tabstop>
  <tabstop>pigtailPrimersCheckBox</tabstop>
  <tabstop>cagTagCheckBox</tabstop>
  <tabstop>m13rCheckBox</tabstop>
  <tabstop>cancelButton</tabstop>
  <tabstop>customTagLineEdit</tabstop>
  <tabstop>useMultiprocessingCheckBox</tabstop>
  <tabstop>repeatsCheckBox</tabstop>
  <tabstop>combinedRepeatsCheckBox</tabstop>
  <tabstop>primersCheckBox</tabstop>
  <tabstop>taggedPrimersCheckBox</tabstop>
  <tabstop>commaSeparatedRadioButton</tabstop>
  <tabstop>tabDelimitedRadioButton</tabstop>
  <tabstop>spinBox_4</tabstop>
  <tabstop>spinBox_6</tabstop>
  <tabstop>spinBox_7</tabstop>
  <tabstop>lineEdit_2</tabstop>
  <tabstop>spinBox_5</tabstop>
  <tabstop>spinBox_8</tabstop>
  <tabstop>spinBox_9</tabstop>
  <tabstop>perfectRepeatsCheckBox</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>runButton</tabstop>
  <tabstop>customTagCheckBox</tabstop>
  <tabstop>hexanucSpinBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>msatcommander</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>217</x>
     <y>525</y>
    </hint>
    <hint type="destinationlabel">
     <x>260</x>
     <y>272</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>runButton</sender>
   <signal>clicked()</signal>
   <receiver>msatcommander</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>457</x>
     <y>525</y>
    </hint>
    <hint type="destinationlabel">
     <x>260</x>
     <y>272</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>primersCheckBox</sender>
   <signal>clicked()</signal>
   <receiver>msatcommander</receiver>
   <slot>checkPrimersOutput()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>398</x>
     <y>251</y>
    </hint>
    <hint type="destinationlabel">
     <x>260</x>
     <y>272</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>taggedPrimersCheckBox</sender>
   <signal>clicked()</signal>
   <receiver>msatcommander</receiver>
   <slot>checkTaggedPrimersOutput()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>398</x>
     <y>271</y>
    </hint>
    <hint type="destinationlabel">
     <x>260</x>
     <y>272</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>combinedRepeatsCheckBox</sender>
   <signal>clicked()</signal>
   <receiver>msatcommander</receiver>
   <slot>checkCombineRepeatsOutput()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>398</x>
     <y>231</y>
    </hint>
    <hint type="destinationlabel">
     <x>260</x>
     <y>272</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>repeatsCheckBox</sender>
   <signal>clicked()</signal>
   <receiver>msatcommander</receiver>
   <slot>checkRepeatsOutput()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>398</x>
     <y>211</y>
    </hint>
    <hint type="destinationlabel">
     <x>260</x>
     <y>272</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
